pcsx2/SPU2/Windows/Config.cpp

```diff
@@ -154,7 +154,9 @@ void ReadSettings()
 	if (mods[OutputModule] == nullptr)
 	{
 		// Unsupported or legacy module.
-		fwprintf(stderr, L"* SPU2: Unknown output module '%s' specified in configuration file.\n", omodid);
+		Console.Warning("* SPU2: Unknown output module '%s' specified in configuration file.\n", omodid);
+		Console.Warning("* SPU2: Defaulting to XAudio (%S).\n", XAudio2Out->GetIdent());
```

Comment:
Is new line at the end needed? Console already does that.

Reply:
good catch!